README.md

```diff
@@ -29,6 +29,23 @@ eslint:{
     }
 }
 ```
+
+## ES6/JSX
+
+We use [ESLint](http://eslint.org/) to lint ES6 and React/JSX code. If ESLint is installed in a project, we can use the configuration from this module by creating a file in the project root named `.eslintrc.yml`. If the project does not use JSX, the file contents should be
+
+```yaml
+extends:
+  - 'node_modules/mobify-code-style/es6/mobify-es6.yml'
+```  
+
+and for a React/JSX project:
+
+```yaml
+extends:
+  - 'node_modules/mobify-code-style/es6/mobify-es6-react.yml'
```

Comment:
For these, I had to add a `./` to the front of the path. So: `./node_modules/mobi.....`